pkg/build/strategies/sti/postexecutorstep.go

```diff
@@ -115,18 +115,18 @@ func (step *commitImageStep) execute(ctx *postExecutorStepContext) error {
 	ctx.labels = createLabelsForResultingImage(step.builder, step.docker, step.image)
 
 	// Set the image entrypoint back to its original value on commit, the running
-	// container has "env" as its entrypoint and we don't want to commit that.
+	// container has "env" as its entrypoint and we don't want to commit that
```

Comment:
Why remove the punctuation? In multiple other places we're adding a missing punctuation, it's inconsistent to remove it here.